data/human/hails.txt

```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
+		"half a"
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" million credits on "
+	word
+		"Ada"
+		"Calda"
+		"Farseer"
+		"Geminus"
+		"Glory"
+		"Hestia"
+		"Mainsail"
+		"Martini"
+		"Pearl"
+		"Vail"
+		"Vinci"
+	word
+		" "
+	word
+		"blackjack."
+		"poker."
+		"backgammon."
+		"baccarat."
+		"roulette."
+
+phrase "friendly civilian"
+	word
+		"Have you read the latest edition of "
+	word
+		"Cosmic Chronicles"
+		"Daring Discoveries"
+		"Norn Nebula News"
+		"Southern Rim Spacer News"
+		"Stars and Starships"
+		"Sunracer Stellar Spectacles"
+		"Tarazed Today"
+		"the Galactic Gazette"
+		"the New Moonshake"
+		"the Satellite Sentinel"
+	word
+		"? "
+	word
+		"It says"
+		"It claims"
+	word
+		" that "
+		" "
+	word
+		"pulse showers are good for your quantum pores"
+		"exposing yourself to neodymium increases your risk of spacepox"
+		"shoddily grown ion algae exhausts your psychic karma"
+		"laser acupuncture lets you communicate with nebula ghosts"
+		"star crystals bring luck to long distance relationships"
+		"consuming moon dust accelerates hair growth but causes lunacy"
+		"excessive exposure to zero gravity can lead to phantom limb syndrome"
+		"alien prosthetics often come with unexpected telepathic abilities"
+		"stardate tattoos increase your chances of alien abduction"
+	word
+		"."
+		"!"
+
+phrase "friendly civilian"
+	word
+		"I'm no weapons expert, but it feels better firing "
+	word
+		"energy blasters"
+		"particle cannons"
+		"proton guns"
+	word
+		" than "
+	word
+		"beam lasers."
+		"heavy lasers."
+
+phrase "friendly civilian"
+	word
+		"I shouldn't have sold my old ship. It was just a "
+	word
+		"Shuttle"
+		"Star Barge"
+		"Sparrow"
+		"Heavy Shuttle"
+		"Freighter"
+		"Fury"
+	word
+		" but it never had "
+	word
+		"stalling engines."
+		"unstable shielding."
+		"overheating fuel cells."
+		"a glitchy HUD."
+		"faulty ship detection."
+
+phrase "friendly civilian"
+	word
+		"Finished cleaning my cargo hold"
+		"Cargo hold is all spick and span"
+		"Finished scrubbing the hold"
+	word
+		". The "
+	word
+		"freshener"
+		"perfume"
+		"spray"
+		"fragrance"
+	word
+		" I used makes it smell like "
+	word
+		"strawberries"
+		"bananas"
+		"peaches"
+		"roses"
+		"jasmine"
+		"lavender"
+	word
+		"."
+		"!"
+
+phrase "friendly civilian"
+	word
+		"Have you seen the holofilm where the "
+	word
+		"good guy"
+		"hero"
+		"bad guy"
+		"villain"
+	word
+		" "
+	word
+		"mounts overcharged twin blasters on a modded Argosy"
+		"shoots up a security station with dual laser pistols"
+		"dodges quad blaster turrets in a stolen Flivver"
+	word
+		"? So cool."
+
+phrase "friendly civilian"
+	word
+		"Frankly, I'm thinking of joining the Navy"
+		"Honestly, I'm going to join the Navy"
+		"I'm enlisting in the Navy"
+	word
+		" "
+	word
+		"because of their dental insurance."
+		"since they'll cover my tuition."
+		"to see if I can get my hands on one of their warships."
+		"because they offer childcare assistance."
```

Comment:
```suggestion
		"because of their dental insurance"
		"since they'll cover my tuition"
		"to see if I can get my hands on one of their warships"
		"because they offer childcare assistance"
	word
		"."
```